tools/nl/embeddings/run_custom.sh

```diff
@@ -15,10 +15,10 @@
 
 set -e
 
+cd ../../..
 python3 -m venv .env
 source .env/bin/activate
-python3 -m pip install --upgrade pip
 pip3 install torch==2.2.2 --extra-index-url https://download.pytorch.org/whl/cpu
-pip3 install -r requirements.txt
+pip3 install -r tools/nl/embeddings/requirements.txt
 
-python3 build_custom_dc_embeddings.py "$@"
\ No newline at end of file
+python3 -m tools.nl.embeddings.build_custom_dc_embeddings "$@"
```

Comment:
This script is not used anywhere. So we can delete it. If you do, please update the commands in the doc: https://github.com/datacommonsorg/website/blob/master/tools/nl/embeddings/build_custom_dc_embeddings.md

Also, can you run these commands as modules to verify that the functionality works as expected: https://github.com/datacommonsorg/website/pull/4216#discussion_r1595836596

Reply:
Verified commands can run. I have also tested with local setup that talks to GCS storage.

Reply:
I found the entire doc uses this script, so will keep as is. As I plan to consolidate the build embedding tool later, can remove at that time.